parity/dapps.rs

```diff
@@ -305,5 +305,10 @@ mod server {
 				})
 				.collect()
 		}
+
+		fn refresh_local_dapps(&self) -> bool {
+			self.endpoints.clone().refresh_local_dapps();
```

Comment:
Except there is a cascading effect. If I don't clone here, I'll have to make it mutable. And then I have to change `rpc::src::v1::helpers::dapps::DappsService` which means all mutable borrows will have to change.

https://doc.rust-lang.org/std/sync/struct.Arc.html

Unless changing the rwlock right away, isn't it standard procedure to clone an arc before use?